packages/client/src/pages/page-editor/page-editor.vue

```diff
@@ -102,365 +101,343 @@ import { collectPageVars } from '@/scripts/collect-page-vars';
 import * as os from '@/os';
 import { selectFile } from '@/scripts/select-file';
 import * as symbols from '@/symbols';
+import { $i } from '@/account';
+import { i18n } from '@/i18n';
+import { MisskeyNavigator } from '@/scripts/navigate';
+
+const nav = new MisskeyNavigator();
+
+const props = defineProps<{
+	initPageId: string,
+	initPageName: string,
+	initUser: string
```

Comment:
All the props are defined as `required: false`
```suggestion
	initPageId?: string,
	initPageName?: string,
	initUser?: string
```